tests/func-tests/virtual_machines_test.go

```diff
@@ -0,0 +1,48 @@
+package tests
+
+import (
+	"flag"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+
+	kubevirtv1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/kubecli"
+)
+
+const timeout = 60 * time.Second
+const pollingInterval = 5 * time.Second
+
+var _ = Describe("Virtual Machines", func() {
+
+	kubecli.Init()
+	flag.Parse()
+	client, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		panic(err)
+	}
+
+	var vmi *kubevirtv1.VirtualMachineInstance
+	var vmiRandName = vmiName + rand.String(48)
+	vmi = kubevirtv1.NewMinimalVMIWithNS(testNamespace, vmiRandName)
+
+	Context("vmi test", func() {
+		It("should create a VirtualMachineInstance", func() {
+			Eventually(func() error {
+				_, err := client.VirtualMachineInstance(testNamespace).Create(vmi)
+				return err
+			}, timeout, pollingInterval).Should(Not(HaveOccurred()), "failed to create a vmi")
```

Comment:
do we want to make sure the VMI comes online as well here? so vmi.Status.Phase = Running?